modules/gapi/src/api/gbackend.cpp

```diff
@@ -143,6 +143,14 @@ void bindInArg(Mag& mag, const RcDesc &rc, const GRunArg &arg, HandleRMat handle
         if (handleRMat == HandleRMat::SKIP) return;
         GAPI_Assert(arg.index() == GRunArg::index_of<cv::RMat>());
         bindRMat(mag, rc, util::get<cv::RMat>(arg), RMat::Access::R);
+
+        // FIXME: Here meta may^WILL be copied multiple times!
```

Comment:
`may^WILL` ?

Reply:
You're right, `may^WWILL` is the right writing

Reply:
Fixed